apps/workflows/tests/test_models.py

```diff
@@ -298,6 +305,7 @@ def test_has_trackers(self, affectedness, resolution, trackers_required):
 
         affect = AffectFactory(
             flaw=flaw,
+            impact=impact,
```

Comment:
Hard-code the parameter here instead of providing a parametrization which is actually the same all the time